docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -14,6 +14,129 @@ then enclave binaries. That information, along with a step-by-step tutorial
 on building an enclave application, is covered in [Getting Started with
 Open Enclave](GettingStarted.md) document.
 
+Using an ACC VM via oe-engine
+------------------------------
+
+The quickest way to begin development would be to use a pre-deployed acc VM. This will install and configure the environment. This document describes performing the deployment on linux. It can also be done on Windows in a similar fashion.
+
+In order to do so, you need to find a binary or build oe-engine, which is located [here](https://github.com/Microsoft/oe-engine).
```

Comment:
Please split this file out from this PR; @johnkord is setting up a private git repo for docs targeted at MS maintainers.

Reply:
I get a 404 when accessing https://github.com/Microsoft/oe-engine